tracer/src/Datadog.Trace/AppSec/Waf/IContext.cs

```diff
@@ -5,11 +5,11 @@
 
 using System;
 using System.Collections.Generic;
+using Datadog.Trace.AppSec.Waf;
 
-namespace Datadog.Trace.AppSec.Waf
+#nullable enable
```

Comment:
You probably didn't mean to move this to the root namespace did you?

```suggestion
#nullable enable

namespace Datadog.Trace.AppSec.Waf;
```

Reply:
good catch 🤦 fixed in d8d7768a2123c01e09b48abf8102f178a4b9c645